### PR TITLE
Initialize DB

### PR DESCRIPTION
**Pull request by NicolasPennie**
Add base setup, ORM, and database tooling. Deployed locally.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[workspace]
+members = [
+    "lib/migration"
+]
+
+[package]
+name = "photon"
+version = "0.1.0"
+edition = "2021"
```
Comment (niks3089) [inline Cargo.toml:9]: why is the edition, 2021? :)

**Review by niks3089 — COMMENTED** [review]
Can we add a separate licence page also?

File: Cargo.toml (hunk 1 above)
Comment (NicolasPennie) [reply Cargo.toml]: fixed

**Comment by NicolasPennie** [comment]
> Can we add a separate licence page also?

license is already in the repo

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+members = [
+    "lib/migration"
+]
+
+[package]
+name = "photon"
+version = "0.1.0"
+authors = ["Helius Labs"]
+homepage = "https://helius.dev"
+license = "AGPL-3.0"
```
Comment (niks3089) [inline Cargo.toml:11]: Licence should be apache?

**Review by niks3089 — APPROVED** [review]

**Pull request merged**